From 37d9e6d926e6805f92b270c5ab843611f22f4da4 Mon Sep 17 00:00:00 2001
From: Wenlin Kang <wenlin.kang@windriver.com>
Date: Mon, 24 Jan 2022 09:16:24 +0000
Subject: [PATCH] drm: xlnx: mixer: remove some unused variable

Remove some unused variable to fix -Wunused-variable warning:

tmp-glibc/work-shared/xilinx-versal/kernel-source/drivers/gpu/drm/xlnx/xlnx_mixer.c:2282:52: warning: unused variable 'port' [-Wunused-variable]
tmp-glibc/work-shared/xilinx-versal/kernel-source/drivers/gpu/drm/xlnx/xlnx_mixer.c:2282:29: warning: unused variable 'vtc_node' [-Wunused-variable]
tmp-glibc/work-shared/xilinx-versal/kernel-source/drivers/gpu/drm/xlnx/xlnx_mixer.c:2695:22: warning: unused variable 'mixer_hw' [-Wunused-variable]

[WK: These changes are already present in the upstream master/xlnx_rebase_v5.10
 branches([1] and [2]) of linux-xlnx repo, but our BSP is based on
 xlnx_rebase_v5.4 branch.
]
Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>
---
 drivers/gpu/drm/xlnx/xlnx_mixer.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/drivers/gpu/drm/xlnx/xlnx_mixer.c b/drivers/gpu/drm/xlnx/xlnx_mixer.c
index c40018dd8760..bb0f6eb89153 100644
--- a/drivers/gpu/drm/xlnx/xlnx_mixer.c
+++ b/drivers/gpu/drm/xlnx/xlnx_mixer.c
@@ -2279,7 +2279,7 @@ static int xlnx_mix_parse_dt_logo_data(struct device_node *node,
 
 static int xlnx_mix_dt_dp_bridge(struct device *dev, struct xlnx_mix *mixer)
 {
-	struct device_node *node, *vtc_node, *disp_node, *port;
+	struct device_node *node, *disp_node;
 
 	node = dev->of_node;
 	/* Disp Bridge support */
@@ -2692,7 +2692,6 @@ static void xlnx_mix_crtc_dpms(struct drm_crtc *base_crtc, int dpms)
 	struct drm_display_mode *mode = &base_crtc->mode;
 	struct drm_display_mode *adjusted_mode =
 					&base_crtc->state->adjusted_mode;
-	struct xlnx_mix_hw *mixer_hw = &mixer->mixer_hw;
 
 	DRM_DEBUG_KMS("dpms: %d\n", dpms);
 	if (mixer->dpms == dpms)
-- 
2.34.1

